_pages/dev/cors.md

```diff
@@ -0,0 +1,164 @@
+# Cross-Origin Resource Sharing (CORS)
+
+While CORS is a standard mechanism on the web, it's considered rather complicated and blocks new Spartacus developer often during their first setup. This is why we spend an article on explaing _how_ CORS affects Spartacus, and _what_ is required to enable the right CORS setup.
```

Comment:
I've seen that most people don't really know what CORS is. Perhaps add a bit more of explanation about it or some links?